wazimap_ng/urls.py

```diff
@@ -149,6 +149,11 @@ def trigger_error(request):
         name="all-details"
     ),
 
+    path(
+        "api/v1/indicator_data/profile/<int:profile_id>/geography/<str:geography_code>/",
+        cache(general_views.consolidated_versioned_data),
+        name="all-details"
```

Comment:
this has the same name as the previous url

Reply:
Will change that right now